modules/search/quora.py

```diff
@@ -84,9 +83,4 @@ def module_api(self):
 	return output
 
 def module_run(self):
-	output = module_api(self)
-	for item in output['links']:
-		title, link = item[1], item[0]
-		self.output(title)
-		self.output(f"\t{link}", 'G')
-		print('')
+	self.alert_results(module_api(self))
```

Comment:
Back it to the previous code. It shows the title and link in a separate portion.

Reply:
Done